samples/RewriteSample/Startup.cs

```diff
@@ -33,10 +39,22 @@ public static void Main(string[] args)
                     options.Listen(IPAddress.Loopback, 5000);
                     options.Listen(IPAddress.Loopback, 5001, listenOptions =>
                     {
-                        // Configure SSL
+                        // Configure HTTPS
                         listenOptions.UseHttps("testCert.pfx", "testPassword");
                     });
                 })
+                .ConfigureAppConfiguration((builderCtx, configurationBuilder) =>
+                {
+                    configurationBuilder.AddInMemoryCollection(new Dictionary<string, string>
+                    {
+                        ["Microsoft:AspNetCore:RedirectToHttps:Port"] = "5001"
```

Comment:
Why not just add a real config file to the sample? This kind of hack only makes sense in a test.